Content.Server/Nutrition/EntitySystems/SliceableFoodSystem.cs

```diff
@@ -118,16 +118,39 @@ public EntityUid Slice(EntityUid uid, EntityUid user, SliceableFoodComponent? co
             return sliceUid;
         }
 
-        private void DeleteFood(EntityUid uid, EntityUid user)
+        private void DeleteFood(EntityUid uid, EntityUid user, FoodComponent foodComp)
         {
             var ev = new BeforeFullySlicedEvent
             {
                 User = user
             };
             RaiseLocalEvent(uid, ev);
+            if (ev.Cancelled)
+                return;
+
+            if (string.IsNullOrEmpty(foodComp.Trash))
+            {
+                QueueDel(uid);
+                return;
+            }
+
+            // Locate the sliced food and spawn its trash
+            var position = Transform(uid).Coordinates;
+            var finisher = Spawn(foodComp.Trash, position);
+
+            var inCont = _containerSystem.IsEntityInContainer(uid);
+            if (inCont)
+            {
+                _handsSystem.PickupOrDrop(user, finisher);
+            }
+            else
+            {
+                var xform = Transform(finisher);
+                _containerSystem.AttachParentToContainerOrGrid((finisher, xform));
+                xform.LocalRotation = 0;
+            }
```

Comment:
Aren't there helper methods on transformsystem to avoid having to manually fix transforms.

Reply:
I'll be honest I pretty much stole that snippet from FoodSystem, so whatever change happens here should happen there too

Reply:
update: turns out i actually stole it from the method immediately above and not FoodSystem, oops brain go fizzle

investigating